docs/contributing/documentation_guidelines.md

```diff
@@ -217,6 +217,24 @@ The reference can be linked to, with an optional label, using the Markdown link
 
 You can read more about other methods of linking to, and creating references in the MyST parser docs section on [Targets and cross-referencing](https://myst-parser.readthedocs.io/en/stable/syntax/syntax.html#targets-and-cross-referencing).
 
+#### Intersphinx links (external docs)
+
+Due to the large amount of documentation links to Django's Sphinx documentation, we have added the integration with this via intersphinx references.
+
+```md
+You can select widgets from [Django's form widgets](inv:django#ref/forms/widgets)
+```
+
+```rst
+    This parameter allows you to specify a :doc:`Django form widget <django:ref/forms/widgets>` to use instead of the default widget for this field type.
+```
+
+There is no support for id (hash) refs at this point, so these will need to be written out as full URLs, remember to use the `stable` URL and not a specific version.
```

Comment:
Is this true? There is such a link in the diff

Reply:
I'll take another look. Tried a bunch of items but maybe I missed something. 

See https://github.com/executablebooks/MyST-Parser/issues/847

Reply:
ahh ok it seems that sometimes you can reference the ID directly (if it exists in the objects that Django docs set up).

Other times you will not be able to.

I will push a PR with an update that tries to communicate this nuance.

Reply:
https://github.com/wagtail/wagtail/pull/11495